options.lua

```diff
@@ -112,6 +113,12 @@ interfaceOptions = {
 					desc = "Use Soulstone/Ankh/etc. resurrection option when one is available and a normal/battle resurrection is not\n\nThis option |cffff0000ignores|r the |cffffd100Don't Accept Revives|r option!",
 					descStyle = "inline", width = "full", type = "toggle", set = optionSetter, get = optionGetter,
 				};
+				dontPostAuctions = {
+					order = 10,
+					name = "|cffffd100Don't Post Auctions|r",
+					desc = "Don't use DialogKey to Post Auctions",
```

Comment:
"Don't allow DialogKey to post auctions"

Reply:
Fixed this